azure-pipelines.yml

```diff
@@ -1,78 +1,79 @@
-#

```

Comment:
This whole file shows as changed because it previously had windows line endings. 😞